uk.co.compucorp.civicrm.hrcore/CRM/HRCore/Upgrader/Steps/1024.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+trait CRM_HRCore_Upgrader_Steps_1024 {
+
+  /**
+   * Removes social account options and reorder de remaining
```

Comment:
There is typo error in `de`. Guess you meant `the`

Reply:
Yes...